frontend/.env

```diff
@@ -0,0 +1 @@
+PUBLIC_BACKEND_URL=http://localhost:8000
```

Comment:
Provide .env.example, with no values.
Add .env in gitignore, never push config values in repo